src/export/digitalPlanning/model.ts

```diff
@@ -254,5 +429,273 @@ export class DigitalPlanning {
+        description:
+          this.passport.data?.[
+            "application.declaration.connection.description"
+          ],
+      },
+    };
+  }
+
+  // @todo support flagsets beyond Planning Permission
+  private getResult(): Payload["result"] {
+    if (this.passport.data?.["application.type"][0].startsWith("pp")) {
+      return [];
+    } else {
+      const result = getResultData({
+        breadcrumbs: this.breadcrumbs as Breadcrumbs,
+        flow: this.flow as LooseFlowGraph,
+      });
+      const { flag } = Object.values(result)[0];
+
+      return [
+        {
+          value: [flag.category, flag.text].join(" / "),
+          description: flag.description,
+        },
+      ] as Payload["result"];
+    }
+  }
+
+  private getProposal(): Payload["data"]["proposal"] {
+    const baseProposal = {
+      projectType: this.passport.data?.["proposal.projectType"].map(
+        (project: string) => {
+          return {
+            value: project,
+            description: this.findDescriptionFromValue("ProjectType", project),
+          };
+        },
+      ),
+      description:
+        this.passport.data?.["proposal.description"] || "Not provided",
+      boundary: this.getBoundary(),
+      date: {
+        start: this.passport.data?.["proposal.start.date"],
+        completion: this.passport.data?.["proposal.completion.date"], // this.passport.data?.["proposal.finish.date"],
+      },
+    };
+
+    if (this.passport.data?.["proposal.extend.area"]) {
+      set(
+        baseProposal,
+        "details.extend.area.squareMetres",
+        this.passport.data?.["proposal.extend.area"],
+      );
+    }
+
+    if (this.passport.data?.["proposal.new.area"]) {
+      set(
+        baseProposal,
+        "details.new.area.squareMetres",
+        this.passport.data?.["proposal.new.area"],
+      );
+      set(
+        baseProposal,
+        "details.new.count.bathrooms",
+        this.passport.data?.["proposal.newBathrooms"],
+      );
+      set(
+        baseProposal,
+        "details.new.count.bedrooms",
+        this.passport.data?.["proposal.newBedrooms"],
+      );
+      set(
+        baseProposal,
+        "details.new.count.dwellings",
+        this.passport.data?.["proposal.newDwellings"],
+      );
+    }
+
+    if (this.passport.data?.["proposal.vehicleParking"]?.length > 0) {
+      set(
+        baseProposal,
+        "details.vehicleParking.type",
+        this.passport.data?.["proposal.vehicleParking"].map(
+          (vehicle: string) => {
+            return {
+              value: vehicle,
+              description: this.findDescriptionFromValue(
+                "VehicleParking",
+                vehicle,
+              ),
+            };
+          },
+        ),
+      );
+
+      if (this.passport.data?.["proposal.vehicleParking"][0] !== "none") {
+        const vehicleTypes = [
+          "cars",
+          "vans",
+          "motorcycles",
+          "bicycles",
+          "buses",
+        ];
```

Comment:
Is there a way we could derive this and `carTypes` from the schema file?

Reply:
This is a good question - all the granular variables are a single list under the `VehicleParking` definition so it'll require a bit of parsing to get the right segments, but totally possible. I've added a `@todo` code comment to pick up in the next round of changes.